src/main/fc/rc_adjustments.c

```diff
@@ -415,7 +415,156 @@ static int applyStepAdjustment(controlRateConfig_t *controlRateConfig, uint8_t a
+        blackboxLogInflightAdjustmentEvent(ADJUSTMENT_PITCH_I, newValue);
+        if (adjustmentFunction == ADJUSTMENT_PITCH_I) {
+            break;
+        }
+        // fall through for combined ADJUSTMENT_PITCH_ROLL_I
+        FALLTHROUGH;
+    case ADJUSTMENT_ROLL_I:
+        newValue = constrain(value, 0, 200); // FIXME magic numbers repeated in cli.c
+        pidProfile->pid[PID_ROLL].I = newValue;
+        blackboxLogInflightAdjustmentEvent(ADJUSTMENT_ROLL_I, newValue);
+        break;
+    case ADJUSTMENT_PITCH_ROLL_D:
+    case ADJUSTMENT_PITCH_D:
+        newValue = constrain(value, 0, 200); // FIXME magic numbers repeated in cli.c
+        pidProfile->pid[PID_PITCH].D = newValue;
+        blackboxLogInflightAdjustmentEvent(ADJUSTMENT_PITCH_D, newValue);
+        if (adjustmentFunction == ADJUSTMENT_PITCH_D) {
+            break;
+        }
+        // fall through for combined ADJUSTMENT_PITCH_ROLL_D
+        FALLTHROUGH;
+    case ADJUSTMENT_ROLL_D:
+        newValue = constrain(value, 0, 200); // FIXME magic numbers repeated in cli.c
+        pidProfile->pid[PID_ROLL].D = newValue;
+        blackboxLogInflightAdjustmentEvent(ADJUSTMENT_ROLL_D, newValue);
+        break;
+    case ADJUSTMENT_YAW_P:
+        newValue = constrain(value, 0, 200); // FIXME magic numbers repeated in cli.c
+        pidProfile->pid[PID_YAW].P = newValue;
+        blackboxLogInflightAdjustmentEvent(ADJUSTMENT_YAW_P, newValue);
+        break;
+    case ADJUSTMENT_YAW_I:
+        newValue = constrain(value, 0, 200); // FIXME magic numbers repeated in cli.c
+        pidProfile->pid[PID_YAW].I = newValue;
+        blackboxLogInflightAdjustmentEvent(ADJUSTMENT_YAW_I, newValue);
+        break;
+    case ADJUSTMENT_YAW_D:
+        newValue = constrain(value, 0, 200); // FIXME magic numbers repeated in cli.c
+        pidProfile->pid[PID_YAW].D = newValue;
+        blackboxLogInflightAdjustmentEvent(ADJUSTMENT_YAW_D, newValue);
+        break;
+    case ADJUSTMENT_RC_RATE_YAW:
+        newValue = constrain(value, 0, CONTROL_RATE_CONFIG_RC_RATES_MAX);
+        controlRateConfig->rcRates[FD_YAW] = newValue;
+        blackboxLogInflightAdjustmentEvent(ADJUSTMENT_RC_RATE_YAW, newValue);
+        break;
+    case ADJUSTMENT_D_SETPOINT:
+        newValue = constrain(value, 0, 254); // FIXME magic numbers repeated in cli.c
+        pidProfile->dtermSetpointWeight = newValue;
+        blackboxLogInflightAdjustmentEvent(ADJUSTMENT_D_SETPOINT, newValue);
+        break;
+    case ADJUSTMENT_D_SETPOINT_TRANSITION:
+        newValue = constrain(value, 1, 100); // FIXME magic numbers repeated in cli.c
+        pidProfile->setpointRelaxRatio = newValue;
+        blackboxLogInflightAdjustmentEvent(ADJUSTMENT_D_SETPOINT_TRANSITION, newValue);
+        break;
+    default:
+        newValue = -1;
+        break;
+    };
+
+    return newValue;
+}
+
+static uint8_t applySelectAdjustment(adjustmentFunction_e adjustmentFunction, uint8_t position)
```

Comment:
I am wondering if the functions in `applySelectAdjustment()` shouldn't be moved into `applyAbsoluteAdjustment()`. After all they are using absolute adjustments, so this would make it clearer where new functions have to be added.